cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -196,18 +197,15 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         return frontend.expressionHandler.handle(node.value)
     }
 
-    private fun handleAnnAssign(node: Python.AST.AnnAssign): Statement {
-        // TODO: annotations
+    /**
+     * Translates a Python [`AnnAssign`](https://docs.python.org/3/library/ast.html#ast.AnnAssign)
+     * into a [Statement].
+     */
+    private fun handleAnnAssign(node: Python.AST.AnnAssign): AssignExpression {
```

Comment:
Do we need the `simple` property?

Reply:
As far as I read the documentation, we can disregard the contents of `simple`.